sources/osgAnimation/BasicAnimationManager.js

```diff
@@ -135,19 +130,12 @@ define( [
             // animations to start
             this._startAnimations = {};
 
-            this._targets.length = 0;
-            this._targetsMap = {};
-
-            this._targetID.length = 0;
-
-            var i;
-            for ( i = 0; i < this._targetIDByTypes.length; i++ )
-                this._targetIDByTypes[ i ].length = 0;
+            this._resetTargets();
 
             this._activeAnimations = {};
             this._activeAnimationList.length = 0;
 
-            for ( i = 0; i < this._activeChannelsByTypes; i++ )
+            for ( var i = 0; i < this._activeChannelsByTypes; i++ )
                 this._activeChannelsByTypes[ i ].length = 0;
 
```

Comment:
My point on array reset is that in osgjs code, to reset an Array, as in "emptying" it,  we do use inconsistently many things: splice, length=0, = [] 
We should use a "single way" of doing it. (that could be enforced with a code standard document) 

Either we use array.splice(0, array.length) or length = 0, or the array = [], but we should do it consistently ?